make/lib/Awt2dLibraries.gmk

```diff
@@ -617,7 +617,7 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBFONTMANAGER, \
     EXTRA_HEADER_DIRS := $(LIBFONTMANAGER_EXTRA_HEADER_DIRS), \
     WARNINGS_AS_ERRORS_xlc := false, \
     DISABLED_WARNINGS_gcc := sign-compare int-to-pointer-cast \
-        type-limits missing-field-initializers implicit-fallthrough strict-aliasing, \
+        type-limits missing-field-initializers implicit-fallthrough strict-aliasing class-memaccess, \
```

Comment:
Added by me. The code that causing warning are changed in JDK tip, so no class-memaccess in JDK tip, but we need it in JDK11